esm/interface/element.js

```diff
@@ -461,7 +461,7 @@ export class Element extends ParentNode {
           const start = next[START];
           if (isOpened) {
             if ('ownerSVGElement' in start)
-              out.push(' />');
+              out.push('/>');
```

Comment:
this is a breaking change for people generating JSX-like output for files to serve + it might play badly with non-quoted attributes as in `<el test=123/>` ... this is a bad idea and it doesn't solve any issue?